tests/fuzz/wasm-mutator-fuzz/smith_wasm.sh

```diff
@@ -8,51 +8,61 @@
 # 1.check parameter
 if [ ! $1 ]; then
 	echo "Parameter is empty, please enter parameter !"
-    exit 
+    exit
 fi
+EXPECTED_NUM=$1
 
 # 2.check dir
 buildPath="./build"
 corpusPath="$buildPath/CORPUS_DIR"
-if [[ ! -d "$buildPath" ]]; then
-    echo "auto create the build folder !"
-    mkdir build
-else # build Folder exists
-    if [[ -d "$buildPath" ]]; then # CORPUS_DIR exists
-        rm -rf $corpusPath
-    fi
-fi
+rm -rf "${corpusPath}"
+mkdir -p "${corpusPath}"
 
 # 3.change dir
-# cd build && mkdir CORPUS_DIR && cd CORPUS_DIR
-cd build && mkdir CORPUS_DIR && cd CORPUS_DIR
+cd "${corpusPath}"
 
 # 4.generate *.wasm file
-echo "Generate $@ files according to user requirements"
+echo "To generate ${EXPECTED_NUM} Wasm files for each kind as required"
```

Comment:
Had better not add `To`, Just `Generate ${EXPECTED_NUM} Wasm files for each kind as required`, it is the thing that is happening but not to happen in the future, adding To really makes confusing.